install_config/install/deploy_router.adoc

```diff
@@ -498,6 +561,74 @@ $ modshard 3 \
     ROUTE_LABELS='sla=low'
 ----
 
+[[using-namespace-router-shards]]
+=== Using Namespace Router Shards
+
+Each
+link:../../architecture/core_concepts/projects_and_users.html#projects[project has its own namespace].
+The routes for a project can be handled by a selected router by using
+`NAMESPACE_LABELS`.  The router is given a selector for a `NAMESPACE_LABELS`
+label and the project that wants to use the router applies the `NAMESPACE_LABELS`
+label to its namespace.
```

Comment:
Routes are `objects` that are associated with `projects/namespaces`. A router just takes the information from routes and provides a mechanism to `direct` traffic to the appropriate `endpoints` (associated with the `service`).
By default, the router `"services"` (warning: overloaded term here) all the routes in all namespaces.
You can filter this set of routes a router `"services"` with selectors.

The NAMESPACE_LABELS selector allows you to select a set of namespaces that the router should `"service"` routes for.  For example, consider a case where you have dev, staging and test all running on a single installation - and you use labels on the projects to indicate which environment a project belongs to. Assuming you have 10 projects and for the sake of this example, let's say 3 of those projects have a label `deployment:env=dev`, you can select just the dev  projects with `NAMESPACE_LABELS="deployment:env=dev"` and run a router that services only the development environment. 

The ROUTER_LABELS on the other hand allows you to filter routes across one or more namespaces. Meaning counter to the above example of having projects tagged with the environment ... let's assume instead you have 1000 routes in different projects and 42 of those routes were tagged with a label "deployment:type=life". To select just those 42 routes from multiple namespaces/projects, you can then use `ROUTER_LABELS="deployment:type=life"`.

I'll write this up and post it here. 
(Aside: Formatting content (that contains markup inside of it)  in a markup in a comment is going to be a bit painful but let's see how it goes).